app/views/check_errors/confirm_clear_all_answers.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :before_content do %>
+  <% content_for :title, "Are you sure you want to clear all?" %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <h1 class="govuk-heading-xl">
+      <%= content_for(:title) %>
+    </h1>
+    <p class="govuk-body">You've selected <%= @questions_to_clear.count %> answers to clear</p>
```

Comment:
Do we need to have anything that takes into account anything that will be cleared as a result of clearing these? (That sounds really hard to identify, so maybe just a warning?)

Reply:
Like conditional questions? Yeah, maybe a warning, let me check with design